back/src/Guide/Types/Core.hs

```diff
@@ -178,15 +178,15 @@ data Item = Item {
   _itemCreated     :: UTCTime,         -- ^ When the item was created
   _itemGroup_      :: Maybe Text,      -- ^ Item group (affects item's color)
   _itemHackage     :: Maybe Text,      -- ^ Package name on Hackage
+  _itemLink        :: Maybe Url,       -- ^ Link to homepage or something
   _itemSummary     :: MarkdownBlock,   -- ^ Item summary
+  _itemEcosystem   :: MarkdownBlock,   -- ^ The ecosystem section
+  _itemNotes       :: MarkdownTree,    -- ^ The notes section
   _itemPros        :: [Trait],         -- ^ Pros (positive traits)
   _itemProsDeleted :: [Trait],         -- ^ Deleted pros go here (so that
                                        --   it'd be easy to restore them)
   _itemCons        :: [Trait],         -- ^ Cons (negative traits)
-  _itemConsDeleted :: [Trait],         -- ^ Deleted cons go here
-  _itemEcosystem   :: MarkdownBlock,   -- ^ The ecosystem section
-  _itemNotes       :: MarkdownTree,    -- ^ The notes section
-  _itemLink        :: Maybe Url        -- ^ Link to homepage or something
+  _itemConsDeleted :: [Trait]          -- ^ Deleted cons go here
```

Comment:
This permutations for order that consists to Postgres schema.

Reply:
I like changing the order for consistency, but changing the order would require an acid-state migration, which is annoying. Can you leave the order as-is and change the schema order instead?

Reply:
of course